.env.sample

```diff
@@ -27,3 +27,7 @@ BARN_ORDERBOOK_USER=
 BARN_ORDERBOOK_PORT=
 BARN_ORDERBOOK_PASSWORD=
 BARN_ORDERBOOK_DB=
+
+# Latest DB URLS (Post CIP-20)
```

Comment:
We will eventually phase out the long form of this above --- problem is those legacy env vars are used in our deployments.